src/Transformer/ArgumentsTransformer.php

```diff
@@ -166,13 +167,13 @@ public function getInstanceAndValidate(string $argType, $data, ResolveInfo $info
                     }
                 }
             }
+            
```

Comment:
It's not showed here, but the line 170 contains spaces, because of which it fails to pass travis job. No empty lines should contain any spaces.

Reply:
Good catch! Thanks ! Fixed ! 👍